systemtest/src/main/java/io/strimzi/systemtest/enums/UserAuthType.java

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.enums;
+public enum UserAuthType {
```

Comment:
Don't we have it in API module?

Reply:
I'm afraid that there is nothing which we can use - I mean, nothing in some `common` module etc. At least I don't know about some enum or class.